workflow-templates/ci-sbt.yml

```diff
@@ -0,0 +1,49 @@
+# Find full documentation here https://docs.github.com/en/actions/learn-github-actions/workflow-syntax-for-github-actions
+name: CI
+
+on:
+  pull_request:
+
+  # Manual invocation.
+  workflow_dispatch:
+
+  push:
+    branches:
+      - $default-branch
+jobs:
+  CI:
+    runs-on: ubuntu-latest
+
+    # See https://docs.github.com/en/actions/security-guides/automatic-token-authentication#permissions-for-the-github_token
+    permissions:
+      # required by guardian/actions-assume-aws-role
+      id-token: write
+      contents: read
+    steps:
+      - uses: actions/checkout@v2
+
+      # Setup AWS credentials to enable uploading to S3 for Riff-Raff.
+      # See https://github.com/guardian/actions-assume-aws-role
+      - uses: guardian/actions-assume-aws-role@v1
+        with:
+          awsRoleToAssume: ${{ secrets.GU_RIFF_RAFF_ROLE_ARN }}
+
+      # Change this if using Java 11.
+      # Configuring caching is also recommended.
+      # See https://github.com/actions/setup-java
+      - name: Setup Java 8
+        uses: actions/setup-java@v2
```

Comment:
Do we consider specifying the action version explicitly to be best practice? I'd be tempted to use `@main` everywhere so that we don't have to keep these templates up to date, but I don't feel too strongly about it.

Reply:
Use of `@main` isn't recommended - https://github.com/actions/toolkit/blob/master/docs/action-versioning.md.

Reply:
Thanks, that keeps things simple then!